google-home-community.groovy

```diff
@@ -1928,7 +1930,18 @@ private executeCommand_BrightnessAbsolute(deviceInfo, command) {
 
 @SuppressWarnings(['UnusedPrivateMethod', 'UnusedPrivateMethodParameter'])
 private executeCommand_GetCameraStream(deviceInfo, command) {
-    return [[:], [:]]
+    checkMfa(deviceInfo.deviceType, "Display", command)
+    def cameraStreamTrait = deviceInfo.deviceType.traits.CameraStream
+    deviceInfo.device."${cameraStreamTrait.cameraStreamCommand}"()
```

Comment:
Should this command get passed the `command.params.SupportedStreamProtocols` so the device can select a protocol from the list of protocols that the target device supports?

Reply:
Are you thinking on the driver side you select everything the driver device can support, and then only link through the protocol the display device supports during the execute?  

Right now, if you pass a format to google that it cannot display, it just gives you a black screen with "smart home camera" on it (so it gracefully handles the negative case).  I didn't make it more complex since you don't know what the display can handle until you actually call up the camera to display the stream.

Thoughts?

Reply:
The list of protocols that Google sends in `command.params.SupportedStreamProtocols` is the list of protocols that the target display supports.  If we pass that to the camera driver then it can pick one of the protocols that both it and the display support.

Reply:
Working on this.  Will need to add URL input fields for all the supported formats at the driver level.

Reply:
Fixed.  Multiple URL's can be specified in the generic driver which get matched to the display device protocols.